tools/gfx/d3d12/render-d3d12.cpp

```diff
@@ -4462,9 +4464,9 @@ class D3D12Device : public RendererBase
                         footprint.Footprint.Depth =
                             Math::Max(1, (textureSize.depth >> mipLevel)) - srcOffset.z;
                     }
-                    footprint.Footprint.RowPitch = (UINT)D3DUtil::calcAligned(
-                        footprint.Footprint.Width * (UInt)formatInfo.blockSizeInBytes,
-                        (uint32_t)D3D12_TEXTURE_DATA_PITCH_ALIGNMENT);
+
+                    assert(dstRowStride % D3D12_TEXTURE_DATA_PITCH_ALIGNMENT == 0);
```

Comment:
This assert is fine to keep here, but we should add this validation in our debug layer.